src/index.js

```diff
@@ -329,7 +343,13 @@ var reactGA = {
         } else {
           var command = pluginName + ':' + action;
           payload = payload || null;
-          if (payload) {
+          if (actionType && payload) {
+            ga(command, actionType, payload);
+            if (_debug) {
+              log('called ga(\'' + command + '\');');
+              log('actionType: ' + actionType + ' with payload: ' + JSON.stringify(payload));
+            }
+          } else if (payload) {
             ga(command, payload);
```

Comment:
Upon further research, I realized that the plugin can be called with `ga('ec:setAction', 'detail');` (in the original ga way). That would translate to react-ga's `ga.execute('ec', 'setAction', 'detail')` and the 3rd parameter is actually the `actionType` and not the `payload`. So technically this may not be the `payload`, but good thing is that it still works :joy: